pkg/controller/kubedirectorcluster/members.go

```diff
@@ -577,6 +667,8 @@ func handleCreatingMembers(
 				}
 			}
 
+			handleConfigPackage(reqLogger, cr, m)
```

Comment:
There's a few error cases in handleConfigPackage that we need to decide how to handle (rather than only logging the error).

If we return early out of handleConfigPackage, returning an error, then we have two choices of what to do here in that case:

* We could return from here WITHOUT doing setFinalState. This would leave the member in creating state and it will be processed again on next reconciliation.

* Or we could use setFinalState to put the member into config error, and return.

Depends on if we think the possible errors causing this are transient or not. At first glance, seems like they are not transient, so we should probably just put the member into config error state.

(If they happen to be caused by the pod going away, then when the pod comes back we'll go through create-pending and creating states again and try everything again.)